spec/requests/users/sessions_spec.rb

```diff
@@ -24,11 +24,23 @@
   end
 
   describe "POST /users/sign_in" do
-    context "when sign in email has been sent" do
+    let(:login_url_regex) { /http:\/\/localhost:3000\/users\/confirm_sign_in\?login_token=.*/ }
+
+    before do
+      mail = instance_double(ActionMailer::MessageDelivery, deliver_now: true)
```

Comment:
By the way, I can't see why you need this?

Reply:
Good point, calling the original method is a better idea :)